content/new-to-f3/index.md

```diff
@@ -0,0 +1,52 @@
+# New to F3?
+
+Everyone starts off as a "friendly new guy" (FNG) and then gets a slightly silly nickname after completing a workout.
```

Comment:
Content taken mostly from Carpex site, some from St. Louis, and I added a timeline at the bottom.

Reply:
👍